components/WelcomePage.tsx

```diff
@@ -1,10 +1,20 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Text, TouchableOpacity, View } from "react-native";
+import style from "../styles";
 
 export default function WelcomePage() {
   return (
-    <View>
-      <Text>Welcome Page</Text>
+    <View style={style.container}>
+      <Text style={style.textHeaderStyle}>WELCOME TO STATES APP!</Text>
+      <TouchableOpacity style={style.buttonStyle}>
+        <Text style={style.butonTextStyle}>Click on the correct state</Text>
+      </TouchableOpacity>
+      <TouchableOpacity style={style.buttonStyle}>
+        <Text style={style.butonTextStyle}>Enter correct state name</Text>
+      </TouchableOpacity>
+      <TouchableOpacity style={style.buttonStyle}>
+        <Text style={style.butonTextStyle}>Quiz on states</Text>
+      </TouchableOpacity>
```

Comment:
This feels repeated with only the text changing. Create a separate component. Make sure you place that component in a proper hierarchy and structure. Just like the `types` folder. Then use that component everywhere in the project.